_posts/2019-12-20-differential-privacy/differential_privacy.Rmd

```diff
@@ -0,0 +1,470 @@
+---
+title: "Differential Privacy with TensorFlow"
+description: > 
+ Differential Privacy guarantees that results obtained from querying a database are basically independent of the presence in the data of a single individual. Applied to machine learning, we expect that no single training example influences the parameters of the trained model in a substantial way. This post introduces TensorFlow Privacy, a library built on top of TensorFlow, that can be used to train differentially private deep learning models from R.
```

Comment:
Could the first part of that first sentence work as "the results of a database query..."?
I'm not sure how much is mitigating language by choice vs. by definition.

Reply:
oh, sure! will change